ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/storage/flat/FullFlatDbReaderStrategy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.ethereum.bonsai.storage.flat;
+
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.storage.KeyValueStorage;
+
+public class FullFlatDbReaderStrategy extends FlatDbReaderStrategy {
+
+  public FullFlatDbReaderStrategy(
+      final MetricsSystem metricsSystem,
+      final KeyValueStorage accountStorage,
+      final KeyValueStorage codeStorage,
+      final KeyValueStorage storageStorage) {
+    super(metricsSystem, accountStorage, codeStorage, storageStorage);
+  }
+
+  @Override
+  public void clearAccountAndStorageDatabase() {
+    // NOOP
```

Comment:
we cannot because I'm checking all the account flat db and if an account is removed I will detect it and add this account to the list of account to repair.

And after I'm using this list to know what king of account storage I need to heal.